File: pgm/ABC_Visual_Basic/ABC_131_A.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As String = Console.ReadLine()
        Dim flg As Boolean = False
        For i As Integer = 1 To S.Length - 1
            If S(i - 1) = S(i)
                flg = True
                Exit For
            End If
        Next
        Console.WriteLine(If(flg, "Bad", "Good"))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_131_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions

class Program
{
    static void Main()
    {
        string S = Console.ReadLine();
        bool flg = false;
        for (int i = 1; i < S.Length; i++)
        {
            if (S[i - 1] == S[i])
            {
                flg = true;
                break;
            }
        }
        Console.WriteLine(flg ? "Bad" : "Good");
        return;
    }
}